### DISCUSSION TITLE
Unable to add / pair Philips Hue Play again

### DISCUSSION THREAD

**Discussion by HB25** [Koenkk/zigbee2mqtt#10532] [Devices]
Hi all,

I have a new Philips Hue Play light bar (ID 440400982842) which I tried to pair with 1.22.1.  The device was detected, i.e. interview happened, but it was not supported, yet.

1.22.2 now adds support for it but I cannot get the device to re-pair or identify again.   The device is detected with Touchlink _scan_ `{"data":{"found":[{"channel":11,"ieee_address":"0x001788010b438b17"}]},"status":"ok"}`  and it also reacts on Touchlink _reset_  (it flashes 3 times) but I see no reaction at Zigbee2Mqtt side.  

There is no useful log entry besides in `zigbee2mqtt/bridge/response/networkmap` but this could be from the first interview? The HA network map does not show the device and it also does not appear in HA entities.

Removing the device from configuration.yaml did not make a difference. 

Does anybody have some hints how to go on to debug this issue? What else can I do?

Thanks in advance!
Heiko

**Comment by reefland** [comment]
Is the issue is not getting the device into pairing mode?  

Are you able to add them to the Hue mobile app?  Upon removing the device again from the Hue app the devices get factory reset and go into pairing mode.  I have 3 Hue Play units and got each to pair doing this.  I enable joining on Zigbee2MQTT and then remove the device from Hue mobile app, within seconds the device resets to default color and Zigbee2MQTT picks up the device and pairs it.

**Comment by HB25** [comment]
Hello,

I do not have the Hue mobile app. Would it work without other original Philips Hue equipment? I will check it out.

The problem mysteriously solved itself by changing the dongle: I used to use a CC2531-based stick which worked well with everything else. (Even the initial interview with the Hue lightbar worked.)  Now I bought a Sonoff ZigBee 3.0 Stick to see if things change - and it works.
One of my TRADFRI remotes does not want to pair anymore, unfortunately, but the rest is still alive.

Cheers, 
Heiko
